include/rocksdb/cloud/cloud_file_system.h

```diff
@@ -395,6 +396,12 @@ class CloudFileSystemOptions {
   // Default: true
   bool delete_cloud_invisible_files_on_open;
 
+  // Experimental option!
+  // Delay after the invisible file is scheduled to be deleted
+  //
+  // Default: 1 hour
+  std::chrono::seconds invisible_cloud_file_deletion_delay;
```

Comment:
This applies to both invisible and obsolete files, so the name is incorrect. Just make it `cloud_file_deletion_delay`.

Invisible files are files that are in the epoch that doesn't match CM epoch for their file number (i.e. were written by a diverged branch of the world). Obsolete files are files that are visible, but no longer part of the LSM tree.

Reply:
🤦  I always forgot this applies to obsolete files as well. Thanks for pointing it out.